### PR TITLE
Update Search Modules

### PR DESCRIPTION
**Pull request by prakhar-ai**
I'm working on adding additional search modules as mentioned in #81. So far I've added a module for TikTok and I'm working on the other modules.

### CODE REVIEW

File: modules/search/tiktok.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+			if link not in output['videos']:
+				output['videos'].append(link)
+
+
+	self.save_gather(output, 'search/tiktok', query, output=self.options.get('output'))
+	return output
+
+def module_run(self):
+	output = module_api(self)
+	self.alert('usernames')
```
Comment (saeeddhqan) [inline modules/search/tiktok.py]: why not? alert_results?

**Comment by saeeddhqan** [comment]
If it is possible to use the source site, use the source site along with search-engines.

File: modules/search/tiktok.py (hunk 1 above)
Comment (prakhar-ai) [reply modules/search/tiktok.py]: changed to using the alert_results function. Some older search modules don't use that function, should I go ahead and add this in those as well?

**Comment by prakhar-ai** [comment]
All current modules use search-engines, how exactly should I go about querying against the source site itself? Also, I found some search modules like the Quora module to be broken (does not give results for usernames), should I fix them as part of this pull request?

**Comment by saeeddhqan** [comment]
should I fix them as part of this pull request? Yes
how exactly should I go about querying against the source site itself? like this: https://www.tiktok.com/search?q=puppy&lang=en

File: modules/search/quora.py [hunk 2]
```diff
@@ -84,9 +83,4 @@ def module_api(self):
 	return output
 
 def module_run(self):
-	output = module_api(self)
-	for item in output['links']:
-		title, link = item[1], item[0]
-		self.output(title)
-		self.output(f"\t{link}", 'G')
-		print('')
+	self.alert_results(module_api(self))
```
Comment (saeeddhqan) [inline modules/search/quora.py]: Back it to the previous code. It shows the title and link in a separate portion.

Comment (prakhar-ai) [reply modules/search/quora.py]: Done

**Comment by saeeddhqan** [comment]
I think it is better to do it during the summer. shall I merge this PR?

**Comment by prakhar-ai** [comment]
Yes, I think you should merge it. I'll work on other modules during the summer.

**Pull request merged**